webshare/share-simple-manual.html

```diff
@@ -14,7 +14,14 @@
         const url = 'https://www.example.com/some/path?some_query#some_fragment';
         setupManualShareTest(false, 'the title', 'the message', url);
         callWhenButtonClicked(() => navigator.share(
-              {title: 'the title', text: 'the message', url: url}));
+              {title: 'the title', text: 'the message', url: url})).then(
+              result => {
+                // Check that promise resolved with undefined. This is guarded
```

Comment:
The structure of the manual tests is: if it fails in a detectable way, it should officially fail using the testharness framework, otherwise, it should neither visibly pass nor fail. It's just an ordinary web page with instructions for the user. The purpose of this is to avoid false positives.

When used in conjunction with tools/runner, the runner will ask the user whether it passed or failed, so this all makes sense.

The purpose of this if statement is that if the result is undefined, it does nothing (it's up to the user to determine whether it passed). But if the result is not undefined, it's fail because it did the wrong thing.